test/scripts/install_kubectl.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+which kubectl >/dev/null 2>&1
+if [ $? -eq 0 ]; then
+    echo "Kubectl already installed"
+    exit 0
+fi
+
+echo "Installing kubect; v1.30.0"
+
+# Install kubectl
+arch=$(uname -m)
+if [ $arch == "x86_64" ]; then 
+    kube_arch="amd64"
+elif [ $arch == "aarch64" ]; then
+    kube_arch="arm64"
+else
+    echo "Error: no kubectl available for $arch"
+fi
+
+curl -LO "https://dl.k8s.io/release/v1.30.0/bin/linux/$kube_arch/kubectl"
+curl -LO "https://dl.k8s.io/release/v1.30.0/bin/linux/$kube_arch/kubectl.sha256"
```

Comment:
nit: can we make the version a var

Reply:
Should this get added to the kind install script? Maybe check that it exists then install?

Reply:
+1 to an env var :)
```suggestion
curl -LO "https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/${kube_arch}/kubectl.sha256"
```

Reply:
Also, downloading the sha256 and then checking it is not considered very secure (if a bad actor modifies the binary... they can also modify the signature hash). Ideally, you would just store the hashes in this script for the specific version and each architecture... 

Then every time you bumped the version .. you may need to collect all hashes for each arch... a little bit annoying.